src/registrar/admin.py

```diff
@@ -1075,6 +1076,21 @@ def organization_type(self, obj):
     # Table ordering
     ordering = ["name"]
 
+    def changeform_view(self, request, object_id=None, form_url="", extra_context=None):
+        """Custom changeform implementation to pass in context information"""
+        if extra_context is None:
```

Comment:
Its basically like "context" in normal Django pages. When you override their views, to add data to their context you need to append data "extra_context" rather than "context". I think you can still pass in data through context, but if I recall its more difficult to get it to work and/or access it

Reply:
Ok I did some digging post our talk earlier today, and I think this may be best route!

Essentially `extra_context` param is used for generic views to pass additional context data to be rendered + allows you to include extra data that isn't directly related to the model that's also being handled by view which is super handy for this change!